inspirehep/modules/records/mappings/v7/records/experiments.json

```diff
@@ -65,10 +71,12 @@
             "type": "object"
           },
           "subgroup_names": {
-            "type": "keyword"
+            "type": "keyword",
+            "copy_to": ["normalized_subgroups_keyword", "normalized_subgroups_text"]
```

Comment:
* If you just need a different type/analyzer for the same field, you can use `fields` instead on `copy_to` (which is mainly used for combining several fields into one).
* why do you need both a `keyword` and `text` variant with different analyzers/normalizers?

Reply:
https://github.com/inspirehep/inspire-next/pull/4026#discussion_r609691842